server/pbench/bin/pbench-satellite-cleanup

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import sys
+import os
+for line in sys.stdin:
+    name = line.strip('\n')
+    try:
+        os.system('mv TO-SYNC/%s SYNCED' % name)
```

Comment:
We might be able to `shutils` here, see http://stackoverflow.com/questions/8858008/how-to-move-a-file-in-python.